app/jobs/importing/run_daily_imports_job.rb

```diff
@@ -168,6 +168,14 @@ def perform
         create_statistical_matches
         generate_logging_info
 
+        GrdaWarehouse::DataSource.importable.with_deleted.each do |data_source|
```

Comment:
I _think_ we want:
```suggestion
        GrdaWarehouse::DataSource.with_deleted.each do |data_source|
```
We now have the situation where a previously importable data source has been converted to an HMIS data source, and we should probably clean those up as well.  There are some data sources (health for instance) that will never have had imports, but that should be a really quick run.  
Alternatively, we could call it once for the non-importable data sources since they shouldn't be changing.